pyright_reportGeneralTypeIssues.json

```diff
@@ -18,6 +18,9 @@
         "pandas/_testing/__init__.py",
         "pandas/_testing/_hypothesis.py",
         "pandas/_testing/_io.py",
+        # Numba does not like typing.cast
```

Comment:
is there any prospect of this being fixed in numba?  i.e. condition under which we can remove this?

Reply:
https://github.com/numba/numba/issues/5474

Reply:
can you add something like "# TODO(numba#5474): can be removed once numba#5474 is addressed"

Reply:
Looks like we lost the ability to add comments in the pyright json config file.